.gitignore

```diff
@@ -1,3 +1,4 @@
 package-lock.json
 node_modules
 /*Agro_easy
+.eslinttrc.json
```

Comment:
I just saw this but you should remove `.eslinttrc.json` from the gitignore. This should be part of the repo